nservicebus/community/index.md

```diff
@@ -93,6 +93,12 @@ Enables identifying NServiceBus messages, commands, and events by conventions us
 
 Created by [Mauro Servienti](https://milestone.topics.it/).
 
+## [NServiceBus.Router](https://github.com/SzymonPobiega/NServiceBus.Router)
+
+NServiceBus.Router is a universal component that is cross-transport, cross-site and cross-cloud router component for NServiceBus.
+
+Created by [Szymon Pobiega](https://simon-says-architecture.com/).
+
```

Comment:
**Suggestion:**
I'm adding this here, because GitHub didn't allow me to do at the top of the file, but David suggested on Slack to add the removed pages routes to the list on the top. 
With that, every request to those pages will land here.

Reply:
Agreed the redirects should be tweaked.

Reply:
Added the redirects